src/ucan.ts

```diff
@@ -66,7 +66,7 @@ export const WNFS_PREFIX = "floofs"
 export async function build({
   audience,
   issuer,
-  lifetimeInSeconds = 30,
+  lifetimeInSeconds = 900,
```

Comment:
Don't we want UCANs to be short lasting? Like basically a new one per request to prevent replay attacks

Reply:
Yeah true. I'll have to find another way than using `fetch-retry` 🤔 Will revise 👀